### DISCUSSION TITLE
Adding a new model with invoke

### DISCUSSION THREAD

**Discussion by jababda** [AbdBarho/stable-diffusion-webui-docker#454] [Q&A] [answered]
Hi all,

I tried adding a safetensors model into data\StableDiffusion but the invoke UI didn't pick it up? 

I'm also a little lost as to how to use this repo. 
Does this just wrap multiple other implementations that don't natively run in docker? If so do I need to go to the invoke repo rather then asking here?

**Comment by AbdBarho** [comment, marked as answer]
Hi, unfortunately, the invoke ui has its own configurations when it comes to the models.

You have to go through the UI and add models there, you can use the path to the model `/data/StableDiffusion/your-model-name-here`


> Does this just wrap multiple other implementations that don't natively run in docker

Pretty much so

> I'm also a little lost as to how to use this repo.


True, I haven't updated the documentation in a while, and a lot of stuff has changed, I will try to give this a look as soon as I find time.

**Reply by jababda** [reply]
Thanks so much :) I wasn't able to get it to detect models I added into that directory, but I might just be using invoke wrong. I'll play with it a bit to see if there's issues with the files I was working with or its an issue with mounting it :)
